EngineLayer/Mods/Mods.txt

```diff
@@ -277,8 +374,22 @@ MT   Mod
 CF   H-4 C-1 S-1 O1
 //
 ID   DiMethyl
-TG   K or R or N
+TG   K or N
+PP   Anywhere.
+MT   Mod
+CF   H4 C2
+//
+ID   DiMethyl (asymmetric)
+TG   R
+PP   Anywhere.
+NL   ETD:45.0579
+MT   Mod
+CF   H4 C2
+//
+ID   DiMethyl (symmetric)
+TG   R
```

Comment:
this particular case might be problematic (2 different dimethyl mods targeting the same residue). unless you're doing ETD there is no way to distinguish these, which means every PSM will be ambiguous between these two mods for HCD/CID searches

I understand the reasons for separating these into two mods. But I think it might be better to have a single mod Dimethyl with NL 45 or 31 Da maybe? it would avoid (imo unnecessary) ambiguity but would make interpretation of ETD spectra  bit more manual